<launch>
  <arg name="machine" default="localhost"/>
  <arg name="settings_yaml" default="$(optenv GOOGLE_DRIVE_SETTINGS_YAML /var/lib/robot/pydrive_settings.yaml)" />
  <arg name="respawn" default="false" />
  <arg name="node_name" default="gdrive_server" />

  <machine name="localhost" address="localhost"/>

  <node name="$(arg node_name)" pkg="gdrive_ros" type="gdrive_server_node.py"
        output="screen" respawn="$(arg respawn)" machine="$(arg machine)">
    <rosparam subst_value="true">
      settings_yaml: $(arg settings_yaml)
    </rosparam>
  </node>
</launch>
